code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -55,11 +97,15 @@
 	radio.canhear_range = 0
 	radio.recalculateChannels()
 
+	occupant_vis = new(null)
+	vis_contents += occupant_vis
+
 /obj/machinery/atmospherics/components/unary/cryo_cell/Exited(atom/movable/AM, atom/newloc)
-	var/oldoccupant = occupant
+	var/mob/living/oldoccupant = occupant
 	. = ..() // Parent proc takes care of removing occupant if necessary
-	if (AM == oldoccupant)
+	if (oldoccupant && istype(oldoccupant) && AM == oldoccupant)
```

Comment:
Can you turn this into an early return?
Something like `if(AM != oldoccupant || QDELETED(oldoccupant)) return`?
I assume that checking the type is not necessary, since `occupant` can only be a living mob, right?

Reply:
Type check is mostly for good measure. Occupant is `mob` not `living`, so it's more of a future-proofing.
Early returning seems unnecessary since it's only two lines being nested.